Tasks/SonarQubePostTest/PRCA/PostComments-Module.psm1

```diff
@@ -1,3 +1,15 @@
+#region constants
+
+$script:discussionWebApiNS = "Microsoft.VisualStudio.Services.CodeReview.Discussion.WebApi"
```

Comment:
This is a "type accelerator" - the namespace string. Powershell does not have "using" clauses and all types need to be fully qualified.